routes/Api/README.txt

```diff
@@ -0,0 +1 @@
+API routes can be placed here.
```

Comment:
I guess instead of readme it is better to use 
User.php to display user details with auth middleware.